SignUpLogInPage/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>SignUp LogIn Page</title>
```

Comment:
Acho que ficava melhor assim
```suggestion
  <title>SignUp LogIn</title>
```

Reply:
Humm, tem o seu sentido, mas como é o nome do projeto também acho que faz sentido ser assim, mas ok, só por cortesia.

Reply:
(Tanta coisa para comentar, e vai comentar o menos importante, mas ok foi bem apanhado)